Music app

harshvachhani1 opened the pull request.

Commit:
explain basic all use for vue

Commit:
explain basic for vue use
use instances,mount,data(),method and use Vue Directives,
builed reactivity,two way data binding etc

Commit:
add some extra guide

Commit:
add city text-box,use watch option,set timeout,
add key event,input binding

Commit:
build circle select colour and checkbox
create a instance and applied function to select value,
make input box for select circle size,dropdown for select font colour,
and also checkbox to while checked to change background colour

Commit:
refactoring code change mistake.

Commit:
builed mode selector and show answer using with v-if.

Commit:
explain how to print array and object
print index and value use with v-for and applied key.

Commit:
used v-for and data shift and push click with button

Commit:
build card style and css change copy button creat

Commit:
use options lifecycle and explain how to work

Commit:
explain and write how to work with proxy and change data

Commit:
explain component and render example write

Commit:
creating a component

Commit:
create new component,create child component,
applied component style,used SASS component,
describe propes,propes limitation performed,
access parent to chiled component,
Component Events used

Commit:
some file mistake change and properly clear error.

Commit:
explain how to use slot
used slot and how to data use parent to child and also applied name slot how to worked.

Commit:
create dynamic component used with is attribute
also write keep-alive built-in component and work with lifecycle hooks

Commit:
explain animation and transition
this file work with transition classes,how to give name for transition,
make random number generate or remove number,
use cdn for library

Commit:
build quiz app used with yarn

Commit:
explain music app just create basic design and setting

vivek-oneglobal commented on musicApp/src/App.vue:
```diff
@@ -0,0 +1,429 @@
+            </div>
+          </li>
+          <li
+            class="flex justify-between items-center p-3 pl-6 cursor-pointer transition duration-300 hover:bg-gray-50"
```
Use for loop as same html is repeating

vivek-oneglobal commented on musicApp/template/index.html:
```diff
@@ -0,0 +1,471 @@
+          >
+            <div>
+              <a href="#" class="font-bold block text-gray-600">Song Title</a>
+              <span class="text-gray-500 text-sm">Artist Name</span>
```
same here

Commit:
new add validation

Commit:
insert new validaion,submit button event
code easy to understand for create component,use slot property,create login and register file

Commit:
use firebase and store user data and compare
